cdap-docs/_common/common-build.sh

```diff
@@ -279,12 +272,21 @@ function check_includes() {
 }
 
 function download_includes() {
+  local includes_dir=${1}
   # $1 passed is the directory to which the downloaded files are to be written.
   # For an example of over-riding this function, see developer/build.sh
   echo "No includes to be downloaded."
 }
 
-function test_includes () {
+function test_includes_directory() {
+  local includes_dir=${1}
+  if [ ! -d "${includes_dir}" ]; then
+    echo "Creating Includes Directory: ${includes_dir}"
+    mkdir ${includes_dir}
```

Comment:
Also, you're guarding the test against spaces in the path, by putting it in quotes, but not doing the same here. If you know there won't be spaces in the path, drop the quotes in the test. Otherwise, add quotes here.